### PR TITLE
Updated docs for Calendar for 13.0

### PR DESCRIPTION
**Pull request by psatyal**
**Done in this commit:** 
- Restructured the page
- Rewrote most of it (a mix of what was there and new text in line with the documentation for the Team planner)
- Added section on subscribing to calendars

**To do:**
- [x] Add new screenshots
- [x] Review text before asking for a review from Marc and Maya

### CODE REVIEW

**Comment by psatyal** [comment]
Hi @MayaBerd and @marcalcobe,

Update the docs concerning the Calendars module. Most of it is re-written for clarity and consistency with other docs pages (like Team Planner). A quick review would be lovely.

The only thing I'm missing is a link to a doc page explaining calendar tokens (or perhaps personal access tokens in general). Do we/will we  have this?

Cheers
Parimal

File: docs/user-guide/calendar/README.md [hunk 1]
```diff
@@ -2,84 +2,124 @@
...
 You can also display the calendar on the dashboard in the [project overview](../project-overview/#calendar-widget).
 
-## Zen mode
+### Zen mode
+
+You can display the calendar full screen mode by hiding the navigation bar  and sidebar to work free of distractions. Click the **Activate zen mode** button on the top of the calendar. Press the escape key (**Esc**) to exist Zen mode.
+
+## Subscribing to a calendar
+
+OpenProject allows you to subscribe to and access any of your calendars using an external client that supports the .ics format.
```
Comment (marcalcobe) [inline docs/user-guide/calendar/README.md]: Hi @psatyal,

We are always talking about iCalendar format not .ics, it is true that the file generated is ICS but the iCalendar is the URL option to keep it updated itself not the downloaded file in ics. I would talk about iCalendar here.

File: docs/user-guide/calendar/README.md [hunk 2]
```diff
@@ -2,84 +2,124 @@
...
 
-Click Escape (**Esc**) to leave the Zen mode.
+![Subscribe to calendar in the More menu](subscribeToCalendar.png)
 
-![zen-calendar](zen-calendar.png)
+To subscribe to a calendar:
 
-## Project overarching calendar
+1.  Click on the **[⋮]** (more) on the toolbar and select **Subscribe to calendar**.
+2.  In the modal that appears, give this calendar a name. We recommend naming it based on where you will be subscribing to this calendar from ("personal phone" or "work tablet" for example).
```
Comment (marcalcobe) [inline docs/user-guide/calendar/README.md]: @psatyal probably we need to specify that the name used can be only used once.

**Review by marcalcobe — COMMENTED** [review]
Hi @psatyal,

I've added some comments. About the link to access tokens I assume it will be available once the PR (#48851) is merged under: docs/getting-started/my-account/#access-tokens.

File: docs/user-guide/calendar/README.md (hunk 1 above)
Comment (psatyal) [reply docs/user-guide/calendar/README.md]: Ahh good point. I've changed it to:

> OpenProject allows you to subscribe to and access any of your calendars using an external client that supports the iCalendar format (such as Thunderbird, Open-Xchange, Apple Calendar or Google Calendar).

File: docs/user-guide/calendar/README.md (hunk 2 above)
Comment (psatyal) [reply docs/user-guide/calendar/README.md]: Good point. Updated!

**Comment by psatyal** [comment]
Thanks @marcalcobe, I've updated the docs based on your review 👍.

**Comment by psatyal** [comment]
Linking to WP: https://community.openproject.org/wp/49464

File: docs/user-guide/calendar/README.md [hunk 3]
```diff
@@ -2,84 +2,124 @@
...
+
+![Calendar time controls](Calendar-timeControls.png)
 
-The numbers in the instruction below correspond to the numbers in the image above:
+1. You can use the previews/next arrow controls (←, →) in the top left corner to move forwards or backwards in time. 
+2. The **today** button brings you back to the current date.
+3. You can use the month/week toggle on the top right corner to switch between either a month view or a week view. 
 
-1. Edit/change the name of your calendar or add additional calendars. Each calendar can be customized to display work packages from specific project(s). You can further fine-tune the information displayed using the filters.
+When multiple work packages are scheduled or span a single day, they are displayed in a vertical stack.
```
Comment (MayaBerd) [inline docs/user-guide/calendar/README.md]: @psatyal do you mean to say "if multiple work packages are scheduled during the same day" ?

File: docs/user-guide/calendar/README.md [hunk 4]
```diff
@@ -2,84 +2,124 @@
...
 
-2. Change your calendar to be private to you or public to your organization: Select the **[⋮] button** -> **Visibility settings**
+Click on a work package strip to view the work package in split screen mode.
```
Comment (MayaBerd) [inline docs/user-guide/calendar/README.md:60]: @psatyal  should we maybe add a sentence that a double click will open the full view?

**Review by MayaBerd — APPROVED** [review]
@psatyal added some minor fixes in a commit and added a couple of comments. Looks good, thank you.

File: docs/user-guide/calendar/README.md (hunk 3 above)
Comment (psatyal) [reply docs/user-guide/calendar/README.md]: Indeed. Your phrasing is clearer, let's use that one 👍

File: docs/user-guide/calendar/README.md (hunk 4 above)
Comment (psatyal) [reply docs/user-guide/calendar/README.md:60]: @MayaBerd I don't think it does, does it?

**Comment by psatyal** [comment]
Hi @MayaBerd, thanks for the review and suggestions! It should be okay to merge now.

File: docs/user-guide/calendar/README.md (hunk 4 above)
Comment (MayaBerd) [reply docs/user-guide/calendar/README.md:60]: @psatyal you are right! my bad, it only works in a wp table view

**Pull request merged**